### DISCUSSION TITLE
Hello

### DISCUSSION THREAD

**Discussion by Olawale-1** [Jay-study-nildana/FrontEndForStudents#45] [General]
Hi,
I am new to Github

**Comment by Jay-study-nildana** [comment]
@Olawale-1 

I would recommend you check out my coding channel, https://www.youtube.com/channel/UCJJVulg4J7POMdX0veuacXw

And this playlist of videos, about beginners. 

https://www.youtube.com/watch?v=VgUESSJLj9Y&list=PL7SHzFbuSdW0-VhFwVTD2CRzpxkHnwKXm

Thanks

Jay

**Reply by vickyaba** [reply]
@Olawale-1 
Hellow
